src/nccl_ofi_rdma.c

```diff
@@ -1631,11 +1702,9 @@ static int process_pending_reqs(nccl_net_ofi_rdma_ep_t *ep)
  */
 static int ofi_process_cq(nccl_net_ofi_rdma_ep_t *ep)
 {
+	struct fi_cq_tagged_entry cqe_tagged_buffers[cq_read_count];
```

Comment:
unnecessary reordering.

Reply:
Moved it out of habit (reverse christmas tree) when removing the two unused declarations for ofi_process_cq.